desired_distance: 1.0
alpha: 0.5
k_fi: 0.3
